app/views/users/show.html.erb

```diff
@@ -1,8 +1,14 @@
-<header>
 <%= stylesheet_link_tag 'user', media: 'all', 'data-turbolinks-track': 'reload' %>
-</header>
+<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
 <body>
+<nav class="nav">
+    <%= link_to "Add New Post", new_user_post_path(@user.id) %>
+    <%= link_to "Home", root_path %>
+    <%= link_to "Back", :back %>
+</nav>
+
 <%= render @user%>
+
 <div id="bio">
  Bio: <%= @user.bio %>
  </div>
```

Comment:
- [ ] [OPTIONAL] As for the recent posts displayed , users should be able to be redirected whenever they click on them. You did not add that feature on the recent posts , Kindy add them 👍 

```ruby
# In the /views/users/show.html.erb
  <a href ="<%= user_post_path(@user, post)%>">
  <%= render post %>
  </a >
```